.eslintrc

```diff
@@ -1,128 +1,3 @@
-		"no-shadow": "off",
-		"no-use-before-define": ["error", "nofunc"],
-		"no-prototype-builtins": "off",
-		"new-cap": "off",
-		"no-plusplus": ["error", { "allowForLoopAfterthoughts": true }],
-		"import/no-unresolved": "error",
-
-		// ES6
-		"prefer-rest-params": "off",
-		"prefer-spread": "off",
-		"prefer-arrow-callback": "off",
-		"prefer-template": "off",
-		"no-var": "off",
-		"object-shorthand": "off",
-		"vars-on-top": "off",
-
-		// TODO
-		"import/no-extraneous-dependencies": "off",
-		"import/no-dynamic-require": "off",
-		"import/newline-after-import": "off",
-		"no-bitwise": "off",
-		"global-require": "off",
-		"max-len": "off",
-		"no-param-reassign": "off",
-		"no-restricted-syntax": "off",
-		"no-script-url": "off",
-		"default-case": "off",
-		"linebreak-style": "off",
-
-		// "no-multi-assign": "off",
-		// "one-var": "off",
-		// "no-undef": "off",
-		// "max-nested-callbacks": "off",
-		// "no-mixed-requires": "off",
-		// "brace-style": "off",
-		// "max-statements-per-line": "off",
-		// "no-unused-vars": "off",
-		// "no-mixed-spaces-and-tabs": "off",
-		// "no-useless-concat": "off",
-		// "require-jsdoc": "off",
-		// "eqeqeq": "off",
-		// "no-negated-condition": "off",
-		// "one-var-declaration-per-line": "off",
-		// "no-lonely-if": "off",
-		// "radix": "off",
-		// "no-else-return": "off",
-		// "no-useless-escape": "off",
-		// "block-scoped-var": "off",
-		// "operator-assignment": "off",
-		// "yoda": "off",
-		// "no-loop-func": "off",
-		// "no-void": "off",
-		// "valid-jsdoc": "off",
-		// "no-cond-assign": "off",
-		// "no-redeclare": "off",
-		// "no-unreachable": "off",
-		// "no-nested-ternary": "off",
-		// "operator-linebreak": "off",
-		// "guard-for-in": "off",
-		// "no-unneeded-ternary": "off",
-		// "no-sequences": "off",
-		// "no-extend-native": "off",
-		// "no-shadow-restricted-names": "off",
-		// "no-extra-boolean-cast": "off",
-		// "no-path-concat": "off",
-		// "no-unused-expressions": "off",
-		// "no-return-assign": "off",
-		// "no-restricted-modules": "off",
-		// "object-curly-spacing": "off",
-		// "indent": "off",
-		// "padded-blocks": "off",
-		// "eol-last": "off",
-		// "lines-around-directive": "off",
-		// "strict": "off",
-		// "comma-dangle": "off",
-		// "no-multi-spaces": "off",
-		// "quotes": "off",
-		// "keyword-spacing": "off",
-		// "no-mixed-operators": "off",
-		// "comma-spacing": "off",
-		// "no-trailing-spaces": "off",
-		// "key-spacing": "off",
-		// "no-multiple-empty-lines": "off",
-		// "spaced-comment": "off",
-		// "space-in-parens": "off",
-		// "block-spacing": "off",
-		// "quote-props": "off",
-		// "space-unary-ops": "off",
-		// "no-empty": "off",
-		// "dot-notation": "off",
-		// "func-call-spacing": "off",
-		// "array-bracket-spacing": "off",
-		// "object-property-newline": "off",
-		// "no-continue": "off",
-		// "no-extra-semi": "off",
-		// "no-spaced-func": "off",
-		// "no-useless-return": "off"
-	}
-}
+	"extends": "nodebb/lib"
```

Comment:
`nodebb/lib`, not `nodebb`?

Reply:
It can be used either `nodebb` and `nodebb/lib`.